registry/handlers/referrers.go

```diff
@@ -59,13 +64,25 @@ func (h *referrersHandler) GetReferrers(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	// extract the artifactType filter.
 	var annotations map[string]string
 	var artifactTypeFilter string
 	if artifactTypeFilter = r.URL.Query().Get("artifactType"); artifactTypeFilter != "" {
 		annotations = map[string]string{
 			v1.AnnotationReferrersFiltersApplied: "artifactType",
 		}
 	}
+
+	// extract the page size info. Users define page size by using the n
+	// query parameter. The default page size is 100.
+	pageSize, nParseError := strconv.Atoi(r.URL.Query().Get("n"))
+	if nParseError != nil || pageSize < minPageSize || pageSize > maxPageSize {
+		pageSize = maxPageSize
```

Comment:
We may want to make the default size to `10`, which is common.

Reply:
Changed `maxPageSize` to 10.

Reply:
You should have three constants.
```go
constant (
    pageSizeMin     = 1
    pageSizeDefault = 10
    pageSizeMax     = 100
)
```

Reply:
Added the three constants.